README.md

```diff
@@ -78,6 +78,21 @@ See our [release page](https://github.com/iTowns/itowns/releases). Note that
 there isn't a lot of support for older version of iTowns, we highly recommand to
 use the last release everytime.
 
+### Try modifications before they are released
+
+Should you want to try some features or bug fixes that are planned for the next release, we provide
```

Comment:
Should -> If ?

Reply:
Indeed, I just learned that the use of "should" in conditional sentences suggests that the clause it describes is unlikely to happen, which is not the case here :slightly_smiling_face: